README.md

```diff
@@ -48,7 +49,33 @@ simplistika.css is a lightweight, responsive and modern CSS framework.
 - [Copyright and License](#license)
 
 ## <a name="gettingstarted"></a>Getting started
-simplistika.css is available via either NPM (preferred), Yarn, or a CDN.
+simplistika.css is available via either direct installation, NPM, Yarn, or a CDN.
```

Comment:
The direct installation option should be after the CDN option and NPM should remain as the preferred option as many developers still prefer to use package managers on their projects.

Reply:
Direct installation is a very KISS option. Directly getting the file with "wget" command is the easiest one among all. The preference is of the user.

Reply:
ok